tests/subtractions/test_api.py

```diff
@@ -401,10 +403,10 @@ async def test_download_subtraction_files(
         return
 
     fasta_expected_path = (
-            client.app["config"].data_path / "subtractions" / "foo" / "subtraction.fa.gz"
+        client.app["config"].data_path / "subtractions" / "foo" / "subtraction.fa.gz"
     )
     bowtie_expected_path = (
-            client.app["config"].data_path / "subtractions" / "foo" / "subtraction.1.bt2"
+        client.app["config"].data_path / "subtractions" / "foo" / "subtraction.1.bt2"
```

Comment:
Take opportunity to change this. Provides typing.
```suggestion
        get_config_from_app(client.app).data_path / "subtractions" / "foo" / "subtraction.1.bt2"
```